test/cases/parsing/issue-758/index.js

```diff
@@ -0,0 +1,50 @@
+it("should require existing module with supplied error callback", function(done) {
+	require.ensure(['./file'], function(){
+		var file = require('./file');
+		file.should.be.eql("file");
+		done();
+	}, function(error) {});
+});
+
+it("should call error callback on missing module", function(done) {
+	require.ensure(['./missingModule'], function(){
+		require('./missingModule');
+	}, function(error) {
+		error.should.be.instanceOf(Error);
+		error.message.should.be.eql('Cannot find module "./missingModule"');
+		done();
+	});
+});
+
+it("should call error callback on missing module in context", function(done) {
+	(function(module) {
+		require.ensure([], function(){
+			require('./' + module);
+		}, function(error) {
+			error.should.be.instanceOf(Error);
+			error.message.should.be.eql("Cannot find module './missingModule'.");
+			done();
+		});
+	})('missingModule');
+});
+
+it("should call error callback on exception thrown in loading module", function(done) {
+	require.ensure(['./throwing'], function(){
+		require('./throwing');
+	}, function(error) {
+		error.should.be.instanceOf(Error);
+		error.message.should.be.eql('message');
+		done();
+	});
+});
+
+it("should not call error callback on exception thrown in require callback", function(done) {
+	require.ensure(['./throwing'], function() {
+    require('./throwing');
+		throw new Error('message');
+	}, function(error) {
+		error.should.be.instanceOf(Error);
+		error.message.should.be.eql('message');
+		done();
+	});
+});
```

Comment:
It looks like maybe I need to add a `test.filter.js` file in the require.ensure error callback test case directory? If so, I'm trying to determine what that file should contain by looking at other test.filter.js examples, but not totally sure how to proceed...